docs/apache-airflow/howto/docker-compose/docker-compose.yaml

```diff
@@ -44,7 +44,6 @@
 #
 # Feel free to modify this file to suit your needs.
 ---
-version: '3.8'
```

Comment:
One potential exception it is keep only for docker-compose [from documentation](https://airflow.apache.org/docs/apache-airflow/stable/howto/docker-compose/index.html#fetching-docker-compose-yaml)

Reply:
People always can download old compose from previous Airflow versions.